content/tutorials/token-authentication.textile

```diff
@@ -607,7 +607,7 @@ blang[android].
   ...
   dependencies {
       ...
-      compile 'io.ably:ably-android:0.8.7'
+      compile 'io.ably:ably-android:1.0.0'
```

Comment:
Do we want to lock into the .0.0 patch version? At a quick google looks like gradle uses https://ant.apache.org/ivy/history/2.1.0/settings/version-matchers.html, so I think we should be able to specify `1.0+` or `1+` to lock into the minor or major versions resp